packages/graphqlgen-templates/typescript-yoga/src/generated/graphqlgen.ts

```diff
@@ -141,8 +142,7 @@ export namespace PostResolvers {
 export namespace UserResolvers {
   export const defaultResolvers = {
     id: (parent: User) => parent.id,
-    name: (parent: User) =>
-      parent.name === undefined || parent.name === null ? null : parent.name,
+    name: (parent: User) => (parent.name === undefined ? null : parent.name),
```

Comment:
Any reason why `()` is needed here?

Reply:
The parenthesis aren't in the code generator. This must be done by prettier somehow 😕